### ISSUE TITLE
Neither Image nor AsyncImage load webp images. 

### ISSUE THREAD

**Issue by Santiagoal1208** [kivy/kivy#8200]
**Software Versions**
* Python: 3.11
* OS:Windows 10
* Kivy:2.2.0.dev0
* Kivy installation method: pip install kivy[base]

**Describe the bug**
Neither Image nor AsyncImage load webp images. 

**Expected behavior**
It should open the webp file like any other png or jpeg

**To Reproduce**

The following python code and kv reproduces the error, it just needs a webp image, like this one : https://download.havecamerawilltravel.com/sample-images/webp/webp-example.webp

PYTHON:

class Pantalla(Widget):
    
    def __init__(self, **kwargs):
        super(Pantalla,self).__init__(**kwargs)
        resourses=os.listdir("Resourses")
        y=Screen(name="hello")
        z=Image(source='webp-example.webp')
        y.add_widget(z)
        self.ids.screen_manager.add_widget(y)
        self.ids.screen_manager.current = "hello"

class PorFavor(MDApp):

    def build(self):
        Builder.load_file("editor.kv")
        return Pantalla()

if __name__ =="__main__":
    PorFavor().run()

KV:

<Pantalla>
    size:root.width,root.height
    Screen:
        size:root.width,root.height
        BoxLayout:
            orientation: 'vertical'
            MDTopAppBar:
                title: "MDToolbar"
                left_action_items: [["menu", lambda x: nav_draw.set_state()]]
            Widget:
        MDNavigationLayout:    
            ScreenManager:
                id: screen_manager
                Screen:
                    id: pantalla_principal
                    name: "scr1"
                    ScrollView:
                        MDList:
                            pacing: dp(300)
                            padding: dp(300)
                            id:grilla

**Code and Logs and screenshots**
Error with AsyncImage
![image](https://user-images.githubusercontent.com/110860294/229631460-cb68f7f2-b5df-4e40-9e39-422e3d8d4d47.png)

Error with Image
![image](https://user-images.githubusercontent.com/110860294/229631665-103783d5-24ab-4e48-97ac-96a996f4d9b6.png)

**Additional context**
I have tried the code on a mac and it works fine there.

**Comment by kuzeyron** [comment]
What happens if you rename `cap 1` to `cap_1`? So there are no space in between.

**Comment by Santiagoal1208** [comment]
Same error.

I looked somewhere else and MAYBE the problem is that it tries to open the image with SDL2 which I don´t know if supports webp, it should be able to open it with PIL but i don´t know why it doesn´t try.

If there is a wayto make PIL the default when opening images?

**Comment by misl6** [comment]
The latest nightly wheels for `macOS` and `Linux` support loading `webp` images via `SDL2`. (as it should)

Will test in a while on `Windows` .

**Comment by misl6** [comment]
Confirmed: On `Windows` `SDL2` fails to load `webp` images.

**Comment by misl6** [comment]
This issue will be fixed via https://github.com/kivy/kivy-sdk-packager/pull/95.

TL;DR; for the future:
- `libwebp` and `libtiff` are shipped into a folder called `optional`.
- These libs need to be moved into the root folder before packaging the `sdl2-deps`

**Comment by misl6** [comment]
Fixed via https://github.com/kivy/kivy-sdk-packager/pull/95 and https://github.com/kivy/kivy/pull/8479
